### ISSUE TITLE
[BUG] failed to register layer: Error processing tar file(exit status 1): Container ID 12011476 cannot be mapped to a host ID

### ISSUE THREAD

**Issue by kaaloo** [opensearch-project/opensearch-build#335]
**Describe the bug**
Pulling the opensearch 1.0.0 image fails with the following error

```bash
❯ docker pull opensearchproject/opensearch:1.0.0
1.0.0: Pulling from opensearchproject/opensearch
99a21848b482: Pull complete 
f4eb0e854a62: Extracting [==================================================>]  432.5MB/432.5MB
4f4fb700ef54: Download complete 
ffd3f715ed97: Download complete 
d19997574184: Download complete 
6f1e5d14588b: Download complete 
3b0e6bcbd287: Download complete 
failed to register layer: Error processing tar file(exit status 1): Container ID 12011476 cannot be mapped to a host ID
```

**To Reproduce**
1. Run `docker pull opensearchproject/opensearch:1.0.0`
2. See error

**Expected behavior**
`docker pull` should succeed

**Plugins**
N/A

**Screenshots**

**Host/Environment (please complete the following information):**
 - OS: Ubuntu
 - Version: 21.04

**Additional context**
N/A

**Comment by solacelost** [comment]
Can confirm that the 1.0.1 tag fixes this issue for me.

**Comment by bbarani** [comment]
@solacelost I am closing this issue as its already fixed in 1.0.1 release. Please feel free to re-open it if you have any other questions.
